en/engine/renderable/model-component.md

```diff
@@ -0,0 +1,57 @@
+
+After batching, the original transform of `ModelComponent` cannot be changed, but the transform of the root node after batching can be changed. Only nodes that meet the following conditions can be statically batched:
+
+  * The child node can only contain `ModelComponent`.
+  * The vertex data structure of `Mesh` of `ModelComponent` under child nodes must be consistent.
+  * The material of `ModelComponent` under child nodes must be the same.
+
+## About dynamic batching
+
+The engine currently provides two sets of dynamic batching systems, *instancing batching* and *dynamic merging VB mode batching*. The two methods cannot coexist, and the priority of instancing is greater than that of dynamic merging VB. To enable batching, simply check the `USE_INSTANCING` or `USE_BATCHING` switch in the material used in the model.
+
+  > **Note**: Batching can participate in the frustum culling process normally, but the transparent model cannot be sorted, which will cause the mixing effect to be incorrect. The engine does not explicitly prohibit the approval of transparent objects, and developers can control the trade-offs.
+
+### Instancing batching
+
+The batch through **instancing** is suitable for drawing a large number of dynamic models with the same vertex data. When enabled, drawing will be grouped according to the material and vertex data, and the instanced attributes information will be organized in each group, and then complete the drawing at one time.
+
+For the support and related settings of the skin model, refer to the [Bone Animation Component](../animation/skeletal-animation.md#AboutDynamic-Instancing) documentation.
+
+In addition, instancing also supports custom additional instanced attributes, which can pass more differential data between different instances (such as the difference in appearance of a diffuse color between different characters, or the influence of wind in a large grass field). This requires the support of custom effects. For more detailed instructions, please refer to the [Syntax Guide](../../material-system/effect-syntax.md#Custom-Instanced-Properties) documentation.
+
+### Merge VB batching
+
+Dynamic batching is suitable for drawing a large number of non-skinned dynamic models with low number of faces and different vertex data. When enabled, drawing will be grouped according to the material, and then the vertex and world transformation information will be merged in each frame of each group, and then completed in batches. <sup id="a1">[1](#f1)</sup>.
+
+Operations such as merging vertices per frame introduce a portion of CPU overhead, which is particularly expensive in JavaScript. In addition, it is necessary to remind that the number of __draw calls__ is not as low as possible. <sup id="a2">[2](#f2)</sup>, most good performance is often the result of CPU and GPU load balancing, so when using batch functions, be sure to do more tests to identify performance bottlenecks and do targeted optimization.
```

Comment:
> the number of __draw calls__ is not as low as possible

minimizing the number of draw calls to the extreme doesn't necessarily lead to the best(or even good) performance

> most good performance

optimal performance